src/components/Button.jsx

```diff
@@ -0,0 +1,6 @@
+export function Button ({children, handleClick}) {
```

Comment:
I don't see the need to add this component because it doesn't have any unique features.